parsers/dataField/parser_test.go

```diff
@@ -334,3 +334,41 @@ func TestParseSCDeploy(t *testing.T) {
 		}, res)
 	})
 }
+
+func TestGuardians(t *testing.T) {
+	arguments := createMockArgumentsOperationParser()
+	parser, _ := NewOperationDataFieldParser(arguments)
+
+	t.Run("SetGuardian", func(t *testing.T) {
+		t.Parallel()
+
+		dataField := []byte("SetGuardian")
```

Comment:
In this very example, this is not a built in function call, since it does not have any argument, thus it is an invaliad guardians setting transaction, so it will be a simple move-balance tx with some data field. Is it ok to add the operation is this case?